README.md

```diff
@@ -165,6 +165,15 @@ one of the following values:
     assumed to be file paths to a single file. If `includes` is provided, `source` and `target` are 
     assumed to be directories, and each member of the `includes` list a glob pattern inside the
     `source` directory.
+  - `rename` - If provided, must contain `from` and `to` attributes. When specified, each file
+    is renamed as it is copied, where `from` is treated as a Python regular expression, and `to`
+    as the substitution string. Backreferences are available for capturing groups, e.g.
+    ```yaml
+      rename:
+        from: "(.*)\\.owl"
+        to: "\\g<1>{version}.owl"
```

Comment:
This syntax for the replace is very unfamiliar. Maybe add a link to the documentation?

Reply:
Included link to Python regex replace docs.